xbmc/pictures/GUIWindowSlideShow.cpp

```diff
@@ -1326,7 +1326,7 @@ void CGUIWindowSlideShow::GetCheckedSize(float width, float height, int &maxWidt
 std::string CGUIWindowSlideShow::GetPicturePath(CFileItem *item)
 {
   bool isVideo = item->IsVideo();
-  std::string picturePath = item->GetPath();
+  std::string picturePath = item->GetDynPath();
```

Comment:
I'm not up-to-date with the whole path / dyn path design / approach. Did you make sure that non-UPnP slideshows still work as well?

Reply:
Yeap, tested all is working as before. GetDynPath fallsback to Path if the dynamic path is not defined. We don't really resolve urls for pictures so in all other cases the dynpath is empty.
Even python plugins just add listitems with the path to the final picture url.